binding/HarfBuzzSharp.Shared/FontFunctions.cs

```diff
@@ -0,0 +1,530 @@
+﻿using System;
+
+namespace HarfBuzzSharp
+{
+	internal delegate bool FontExtentsProxyDelegate (IntPtr font, IntPtr fontData, out FontExtents extents,
+		IntPtr context);
+	internal delegate bool NominalGlyphProxyDelegate (IntPtr font, IntPtr fontData, uint unicode,
+		out uint glyph, IntPtr context);
+	internal delegate bool VariationGlyphProxyDelegate (IntPtr font, IntPtr fontData, uint unicode,
+		uint variationSelector, out uint glyph, IntPtr context);
+	internal delegate uint NominalGlyphsProxyDelegate (IntPtr font, IntPtr fontData, uint count,
+		IntPtr firstUnicode, uint unicodeStride, IntPtr firstGlyph, uint glyphStride, IntPtr context);
+	internal delegate int GlyphAdvanceProxyDelegate (IntPtr font, IntPtr fontData, uint glyph, IntPtr context);
+	internal delegate void GlyphAdvancesProxyDelegate (IntPtr font, IntPtr fontData, uint count,
+		IntPtr firstGlyph, uint glyphStride, IntPtr firstAdvance, uint advanceStride, IntPtr context);
+	internal delegate bool GlyphOriginProxyDelegate (IntPtr font, IntPtr fontData, uint glyph, out int x,
+		out int y, IntPtr context);
+	internal delegate int GlyphKerningProxyDelegate (IntPtr font, IntPtr fontData, uint firstGlyph,
+		uint secondGlyph, IntPtr context);
+	internal delegate bool GlyphExtentsProxyDelegate (IntPtr font, IntPtr fontData, uint glyph,
+		out GlyphExtents extents, IntPtr context);
+	internal delegate bool GlyphContourPointProxyDelegate (IntPtr font, IntPtr fontData, uint glyph,
+		uint pointIndex, out int x, out int y, IntPtr context);
+	internal unsafe delegate bool GlyphNameProxyDelegate (IntPtr font, IntPtr fontData, uint glyph, char* name,
```

Comment:
Is there a reason why this is `char*` instead of `Span<T>`? Or even a `string`?

Reply:
I will change it to string. Wasn't sure at the beginning what HarfBuzz is expecting here.

Reply:
I just pushed the changes - let me make some comments